server/schedule/operator/metrics.go

```diff
@@ -24,8 +24,18 @@ var (
 			Help:      "Bucketed histogram of processing time (s) of finished operator step.",
 			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 16),
 		}, []string{"type"})
+
+	// OperatorLimitCounter exposes the counter when meeting limit.
+	OperatorLimitCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "pd",
+			Subsystem: "schedule",
+			Name:      "operator_limit",
+			Help:      "Counter of operator meeting limit",
+		}, []string{"type", "name"})
 )
 
 func init() {
 	prometheus.MustRegister(operatorStepDuration)
+	prometheus.MustRegister(OperatorLimitCounter)
```

Comment:
Maybe it will be more convenient if support the source store and target store.

Reply:
The labels for recording `source` and `target` needs to be tested considering the performance and storage for prometheus as same as https://github.com/tikv/pd/pull/3320